### PR TITLE
homework2

### PR DESCRIPTION
**Pull request by NightNomad**
Случайно залил .DS_Store как лох :(

### CODE REVIEW

File: 2015/Ponomarenko/homework2/gemfiler/lib/main.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+
+class Gemfiler
+
+  def execute
+    args = parse_arguments
+    gem_name = args[:name]
+    version_specs = args[:versions]
```
Comment (bypotatoes) [inline 2015/Ponomarenko/homework2/gemfiler/lib/main.rb]: I'd suggest:
```
gem_name, gem_version = parsed_arguments
```
where parsed arguments returns array
Yr loving parallel assignment)

File: 2015/Ponomarenko/homework2/gemfiler/lib/search&filter.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+class Searcher
+  attr_reader :gem_name
+  attr_accessor :versions
+
+  def initialize(name)
+    @name = name
+    @versions = []
+  end
+
+  def search
```
Comment (Nazarchuk) [inline 2015/Ponomarenko/homework2/gemfiler/lib/search&filter.rb]: def search
     version_list = Gems.versions(@name)
     @versions = version_list.map { |details| details['number'] }
 rescue
    raise 'Cannot found gem'
end

Comment (Nazarchuk) [inline 2015/Ponomarenko/homework2/gemfiler/lib/search&filter.rb]: #def search
### version_list = Gems.versions(@name)
###@versions = version_list.map { |details| details['number'] }
#rescue
###raise 'Cannot found gem'
# end

File: 2015/Ponomarenko/homework2/gemfiler/lib/search&filter.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    begin
+      version_list = Gems.versions(@name)
+    rescue
```
Comment (bypotatoes) [inline 2015/Ponomarenko/homework2/gemfiler/lib/search&filter.rb]: It's a good way to rescue specific error, by default it's StandardError

File: 2015/Ponomarenko/homework2/gemfiler/lib/search&filter.rb (hunk 2 above)
Comment (Nazarchuk) [inline 2015/Ponomarenko/homework2/gemfiler/lib/search&filter.rb]:
```
def search
  version_list = Gems.versions(@name)
  @versions = version_list.map { |details| details['number'] }
rescue
  raise 'Cannot found gem'
end
```
можешь тут так укоротить

**Pull request closed without merging**